File: Molinos.Scato.Dominio/Entidades/ArchivosPuerto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;


namespace Molinos.Scato.Dominio.Entidades
{
    public class ArchivosPuerto : IIdentificable
    {
        [Key]
        public virtual int Id { get; set; }
        public virtual Embarque Embarque { get; set; }
        public virtual int Usuario_Id { get; set; }
        public virtual string NombreArchivo { get; set; }
        public virtual string Archivo { get; set; }
        public virtual TipoArchivoPuerto TipoArchivoPuerto { get; set; }
        public virtual DateTime Fecha { get; set; }

    }
}

File: Molinos.Scato.WebPuerto/src/app/shared/modelos/ArchivosPuerto.ts
import { TipoArchivoPuerto } from "./TipoArchivoPuerto";

export class ArchivoPuerto{
    id: number;
    tipoArchivoPuerto: TipoArchivoPuerto;
    usuario_Id: number;
    embarque_Id: number;
    nombreArchivo: string;
    archivo: string;
    fecha: Date;
}